dao/src/integ-test/resources/uk/gov/moj/sdt/dao/sql/TargetApplicationDaoTest.sql

```diff
@@ -0,0 +1,3 @@
+insert into TARGET_APPLICATIONS (TARGET_APPLICATION_ID, TARGET_APPLICATION_CODE, TARGET_APPLICATION_NAME, VERSION_NUMBER) values (1, '1104', '2', 0);
```

Comment:
Bit of a minor quibble but this test data doesn't quite have the sort of values that are expected.  See https://github.com/hmcts/civil-sdt/blob/master/database/data/create_reference_data.sql for some examples.

Reply:
Corrected